src/DynamoCore/DynamoCore.csproj

```diff
@@ -21,6 +21,8 @@
     <Reference Include="System.Configuration" />
   </ItemGroup>
   <ItemGroup>
+    <PackageReference Include="Lib.Harmony" Version="2.2.1" ExcludeAssets="runtime" Condition="'$(Configuration)' != 'Debug'" />
```

Comment:
so in release builds we exclude the runtime assets, and in debug builds we don't... we need the non debug condition so the build succeeds?

I don't know if this means we should add harmony to the about box, license info etc since even though we don't distribute it in release builds we still pull it.

Reply:
Ugh..you're right. That would need legal clarification.
I rather have devs need to pull it manually than go through another round with legal

Reply:
haha, what about using reflection to call it so theres no compile time dep?

Reply:
the other thing to consider @pinzart90 is that MIT is a preapproved license so I think you can just use it and update the license info without discussion.

Reply:
`about using reflection to call it so theres no compile time dep`
Makes development really tough ...
I updated the about box, as you suggested